pkg/bugzilla/bugzilla.go

```diff
@@ -41,6 +41,33 @@ type pr struct {
 	prNum int
 }
 
+type bugzillaRequestError struct {
+	statusCode   int
+	bugzillaCode int
+	message      string
+}
+
+func (e bugzillaRequestError) Error() string {
+	if e.bugzillaCode != 0 {
+		return fmt.Sprintf("code %d: %s", e.bugzillaCode, e.message)
+	}
+	return e.message
+}
+
+func validateBZRequestError(err error) bool {
+	acceptedReturnCodes := []int{200, 201}
```

Comment:
It's confusing to compare a "returnCode" with a "statusCode"...
This should probably be `acceptedStatusCodes`.

Reply:
Fixed